### DISCUSSION TITLE
"All checks have passed" disappears after commit

### DISCUSSION THREAD

**Discussion by dellow** [community/community#25055] [Actions]
<p>I run my actions when a pull request is opened or reopened.</p>
<p>I have a task that builds some javascript and commits back to the repo using the “stefanzweifel/git-auto-commit-action” action.</p>
<p>The problem is, when the commit is actioned the “All checks have passed” part of the pull request disappears, I assume because the pull request has now changed after the commit.</p>
<p>Is there anyway to avoid this from happening?</p>

**Comment by ibakshay** [comment, marked as answer]
<p><a class="mention" href="/dellow">@dellow</a> I am also facing the same  issue and I already opened a ticket <a href="https://github.community/t/bug-report-failed-status-checks-getting-removed-after-commit-from-the-github-action-bot/17370/2">here</a> </p>

**Comment by dellow** [comment]
<p>Ah, ok I’ll accept your answer and head over to your post. Thanks</p>

**Comment by baran-goru** [comment]
@dellow @ibakshay : I cannot open the referenced ticket. maybe it doesn't exist anymore. can you please comment here if you found a solution? or a workaround?

**Comment by Carles-Figuerola** [comment]
Same as above, the "answer" is not valid anymore as the duplicate question seems to have been removed. This is still happening to us.

**Comment by jeffpeck10x** [comment]
Just found this thread, having the same problem in August, 2024
